examples/03_connectivity/plot_canica_resting_state.py

```diff
@@ -3,7 +3,8 @@
 =====================================================
 
 An example applying CanICA to resting-state data. This example applies it
-to 30 subjects of the ADHD200 datasets. Then it plots a map with all the
+to 30 subjects of the resting-state development datasets (children).
```

Comment:
Are we selecting only children, here ? I thought [from the fetcher](https://github.com/nilearn/nilearn/pull/1953/files#diff-472cc8a2f83316878a7f02c131580611R2081) that it would select both children and adults when given a `n_subjects` value...